backend/director/tools/videodb_tool.py

```
@@ -36,12 +36,27 @@ def get_collections(self):
+        """
+        try:
+            image = self.collection.get_image(image_id)
+            return {
+                "id": image.id,
+                "url": image.url,
+                "name": image.name,
+                "description": getattr(image, "description", None),
+                "collection_id": image.collection_id,
+            }
+        except Exception as e:
+            raise Exception(f"Failed to fetch image with ID {image_id}: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)